corehq/apps/translations/generators.py

```diff
@@ -254,6 +254,7 @@ def _get_translation_for_sheet(self, app, sheet_name, rows):
             unique_id_index = self._get_header_index(MODULES_AND_FORMS_SHEET_NAME, 'unique_id')
 
             def occurrence(_row):
+                sheet_name_index = sheet_name_index.replace("menu", "module")
```

Comment:
the correct change here would be
`return ':'.join([_row[type_index], _row[sheet_name_index].replace("menu", "module"), _row[unique_id_index]])`

Reply:
We would need to keep the type as "Module" so this should be,
`return ':'.join([_row[type_index].replace("menu", "Module"), _row[sheet_name_index].replace("menu", "module"), _row[unique_id_index]])`